packages/flutter/lib/src/widgets/focus_manager.dart

```diff
@@ -911,7 +911,10 @@ class FocusNode with DiagnosticableTreeMixin, ChangeNotifier {
 
   @override
   String toStringShort() {
-    return '${describeIdentity(this)}${debugLabel != null && debugLabel.isNotEmpty ? '($debugLabel)' : ''}';
+    final String extraData = '${debugLabel != null && debugLabel.isNotEmpty ? '$debugLabel ' : ''}'
+        '${hasFocus && !hasPrimaryFocus ? '[FOCUSED]' : ''}'
+        '${hasPrimaryFocus ? '[PRIMARY]' : ''}';
```

Comment:
Wondering if it should still say FOCUSED in this case because it is still focused right?

Reply:
Yeah, but I thought that "[PRIMARY FOCUSED]' was too long for a "short" string. If you don't think so, I'm happy to add it in too.

Reply:
I feel like being more explicit here would be ok. It's still pretty short :)

Reply:
OK, done.